src/core/opamStd.mli

```diff
@@ -510,11 +510,12 @@ module Sys : sig
   (** Queried lazily *)
   val os: unit -> os
 
-  (** The output of the command "uname", with the given argument. Memoised. *)
-  val uname_cmd: string -> string option
+  (** The output of the command "uname -U". FreeBSD only. *)
```

Comment:
Also, it's intended to be used only on freebsd, but no check is done for that. Proposal:
```
  (** FreeBSD version, probed via "uname -U".
      To use only on FreeBSD, otherwise returns [None]. *)

Reply:
i'm not sure about this suggestion. The fact that the function returns the freebsd version is already described in the function name. I prefer my version